src/react-components/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
-### CHANGE LOG
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+When it is released, this project will adhere to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
 ========
-### v1.0.1
-> Add description
+
+## [1.0.2] - 2019-11-13
```

Comment:
So a question for our changelogs and Lerna... do we think we'll reset all of the major numbers on these to "0" when we decide we are ready for our first major release?

Reply:
I don't know.  NPM won't let us remove and re-publish the package, so if we wanted to do something like this we'd have to pick a different package name.

Reply:
Note to self to bump the package.json version numbers

Reply:
Renaming the design system is thus a _necessity_ 

`reservoir` here we come

Reply:
Maybe we keep calling it a 0 version until we move to the permanent name... ?

Reply:
Or maybe we append -prerelease?

Reply:
Added "Prerelease" to the changelogs.  Hopefully this is ... ok for what we need for the time being?

Reply:
I'm fine with it. I would prefer to have a `0.x.x` but we got what we got!

Reply:
If we released under a new package, we could use `0.x.x` - the reason why we weren't doing it was because Lerna treats every pre-1 release as a major release, which ... actually works for us now haha